homeassistant/components/cookidoo/strings.json

```diff
@@ -0,0 +1,447 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Login to Cookidoo",
+        "data": {
+          "email": "[%key:common::config_flow::data::email%]",
+          "password": "[%key:common::config_flow::data::password%]",
```

Comment:
add data_descriptions for all flow data keys

Reply:
@zweckj Even for email and password? Are there at least standard values such as `[%key:common::config_flow::data_description::password%]`?

Reply:
Did not find any in the code, added my own.

Reply:
yes for all of them, no there are no standards